.mocharc.yml

```diff
@@ -0,0 +1,3 @@
+---
+# Configuration options passed to Mocha when running tests.
+timeout: 3000
```

Comment:
Some tests took a bit longer than the default 2s, so I bumped up the timeout for all Mocha tests.